lib/core/src/server/manager/manager-webpack.config.js

```diff
@@ -58,9 +58,44 @@ export default ({ configDir, configType, entries, dll, outputDir, cache, babelOp
       new webpack.DefinePlugin({ 'process.env': stringified }),
       new CaseSensitivePathsPlugin(),
       new Dotenv({ silent: true }),
+      // graphql sources check process variable
+      new DefinePlugin({
+        process: JSON.stringify(true),
```

Comment:
having this set to true is causing me some issues with the latest `@next` release because true does not match the expected shape of `process`, which is normally an object. 

in the case i'm running into, the library checks that `process` exists and then tries to search for the `env` property using the `in` operator, but fails since `in` cannot be used on a boolean. 

if this really needs to be included in the default config, could it at least be set to an object to match the contract of what `process` would normally be?